php-practice/section4.php

```diff
@@ -85,35 +84,36 @@ public function playCampaign() {
  * QUESTION 1
  *
  * What are the 2 main things that you can create inside of a class?
- * ANSWER:
+ * ANSWER: property variables and the constructor
  */
 
 /**
  * QUESTION 2
  *
  * What keyword do you use in PHP to create an object from a class?
- * ANSWER:
+ * ANSWER: new
  */
 
 /**
  * QUESTION 3
  *
  * When you instantiate (create) a new object, which method (function) runs from its class?
- * ANSWER:
+ * ANSWER: __construct method
  */
 
 /**
  * QUESTION 4
  *
  * If the constructor is set up to take arguments, how or where do you specify which arguments to pass in?
- * ANSWER:
+ * ANSWER: I pass the arguments in the () next to the __construct keyword
  */
 
 /**
  * QUESTION 5
  *
  * How do you access a public property of an object?
- * ANSWER:
+ * ANSWER: We acess the public property of an object by echoing out the name of the varible (in this case $starCraft2) followed by -> and then the property we want to access 
+ * which would look like this - echo 'My favorite game is' . $starCraft2->title;
```

Comment:
Good answer.

By "access", I simply meant getting the value. Echoing out the value is an example situation where you would need to get the value. But you could do other things with the value, like...

1. Put it in a database
2. Use it to figure out further logic of your program.

etc...